genfragments/ThirteenTeV/Higgs/HY/ResonanceDecayFilter_example_HY_XToYH_HTo2B_YTo2WTo2L2Nu.py

```diff
@@ -0,0 +1,49 @@
+import FWCore.ParameterSet.Config as cms
+
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
+                                     args = cms.vstring('/afs/cern.ch/work/g/gsaha/public/FragmentCheck_HHbbWW/GridPacks/NMSSM_XToYH_MX_1000_MY_500_slc7_amd64_gcc700_CMSSW_10_6_19_tarball.tar.xz'), ##-- Locally produced gridpack
+                                     nEvents = cms.untracked.uint32(25),
+                                     numberOfParameters = cms.uint32(1),
+                                     outputFile = cms.string('cmsgrid_final.lhe'),
+                                     scriptName = cms.FileInPath('GeneratorInterface/LHEInterface/data/run_generic_tarball_cvmfs.sh')
+                                 )
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+    maxEventsToPrint = cms.untracked.int32(1),
+    pythiaPylistVerbosity = cms.untracked.int32(1),
+    filterEfficiency = cms.untracked.double(1.0),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(13000.),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CP5SettingsBlock,
+        pythia8PSweightsSettingsBlock,
+        processParameters = cms.vstring(
+            '25:onMode = off', # Turn off all H decays
+            '25:oneChannel = 1 1 100 5 -5', # H->b b~
+            '25:onIfMatch = 5 -5',
+            '35:onMode = off',
+            '35:oneChannel = 1 1 100 24 -24',  # Y->WW
+            '35:onIfMatch = 24 -24',
+            '15:onMode = on', # allow all tau decays. Leptonic and Hadronic
+            '24:mMin = 0.05', # the lower limit of the allowed mass range generated by the Breit-Wigner (in GeV)
+            '24:onMode = off', # Turn off all W decays
+            '24:onIfAny = 1 2 3 4 5 11 13 15', # Add W->enu, W->munu, W->taunu. Add W->qq decays
```

Comment:
hi @acarvalh for line35, why not use "24:onIfAny = 11 13 15" since you only consider the leptonic decay, it should more efficient.

Reply:
Thanks for the catch, done in both cards. I tag @gsaha009 in case any more comments